src/Fixer/ClassNotation/OrderedTypesFixer.php

```diff
@@ -308,7 +308,7 @@ private function isTypeSortable(TypeAnalysis $type): bool
      */
     private function collectDisjunctiveNormalFormTypes(string $type): array
     {
-        $types = array_map(static function ($subType) {
```

Comment:
any SCA to prevent adding another not-typed lambdas?

Reply:
I've spotted these with [slam/phpstan-extensions](https://github.com/Slamdunk/phpstan-extensions) but was not able to fix all of the types (some lambdas need multiple types and with PHP 7.4 we cannot have them).

Reply:
@kubawerlos 
maybe for those cases we can add `mixed` typehint, and prevent adding any more untyped lambdas?
then, we deal with `mixed` afterwars

Reply:
Definitely, as soon as we drop PHP 7.4 support.